languages/elixir/exercises/concept/tuples/.meta/design.md

```diff
@@ -0,0 +1,47 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of tuples and common conventions in elixir to return multiple values.
+
+This is done through the implementation of a basic kitchen calculator.
+
+## Learning objectives
+
+- Know what a tuple is
+- Know how to write a tuple literal
+- Know how to return a tuple from a function
+- Know that returning a tuple is a common for of multiple return in Elixir
+- Know how to extract an element from a tuple using `elem/2`
+- Know how to write a floating point literal
+- Know how to round a floating point number to decimal place or integer
```

Comment:
Looking at the example solution, the topic of rounding floats is not covered here. I'm with Erik (https://github.com/exercism/v3/pull/1447/files#r427835478), thinking floats are a prerequisite for this exercise, not really taught in it.